### ISSUE TITLE
Suppress logs with google.cloud.workflows client instantiation

### ISSUE THREAD

**Issue by tlienart** [googleapis/google-cloud-python#12902]
### Determine this is the right repository

- [X] I determined this is the correct repository in which to report this bug.

### Summary of the issue

Using `google-cloud-workflows` version 1.14.4 with python 3.12 and running

```python
from google.cloud import workflows_v1
from google.cloud.workflows import executions_v1

WF_EXEC_CLIENT = executions_v1.ExecutionsClient()
```

shows

```
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1720769762.366880 10627235 config.cc:230] gRPC experiments enabled: call_status_override_on_cancellation, event_engine_dns, event_engine_listener, http2_stats_fix, monitoring_experiment, pick_first_new, trace_record_callops, work_serializer_clears_time_cache, work_serializer_dispatch
I0000 00:00:1720769762.376931 10627235 check_gcp_environment_no_op.cc:29] ALTS: Platforms other than Linux and Windows are not support
```

in a similar fashion,

```python
WF_CLIENT = workflows_v1.WorkflowsClient()
```

shows

```
I0000 00:00:1720769763.515004 10627235 work_stealing_thread_pool.cc:321] WorkStealingThreadPoolImpl::PrepareFork
I0000 00:00:1720769764.069125 10627235 check_gcp_environment_no_op.cc:29] ALTS: Platforms other than Linux and Windows are not supported
```

### Desired behaviour + what I tried

Ability to suppress these logs. I tried several approaches including basic stderr/stdout redirection, importing `abseil-py`, and following suggestions in https://github.com/abseil/abseil-py/issues/99, but was ultimately not successful. 

### API client name and version

google-cloud-workflows v1.14.4

### OS & version + platform

macos sonoma 14.5

### Python environment

python 3.12

**Comment by parthea** [comment]
This issue appears to only happen with recent gRPC release `grpcio==1.65.0` as per https://github.com/grpc/grpc/issues/37162. `grpcio==1.65.0` was 'yanked' from PyPI today (https://pypi.org/project/grpcio/1.65.0/).  Please run `pip install grpcio==1.64.1` to revert back to the last stable version of gRPC. I'm going to close this issue as obsolete, but please open a new issue if you're still seeing excess logs with the latest stable release `grpcio==1.64.1`.

**Comment by adamryczkowski** [comment]
As of `grpcio==1.65.1` the issue still persists for Pythons 3.12 and 3.10

**Comment by aoezdTchibo** [comment]
> As of `grpcio==1.65.1` the issue still persists for Pythons 3.12 and 3.10

Also still in `grpcio==1.65.2`

**Comment by khageshsaini** [comment]
Also still in `grpcio==1.65.4`

**Comment by petemounce** [comment]
I just tried out `grpcio==1.66.0` and this issue looks addressed in that.

**Comment by aoezdTchibo** [comment]
> I just tried out `grpcio==1.66.0` and this issue looks addressed in that.

Issue therefore also resolved in `google-cloud-logging` version 3.11.2 🎉
